webui/webui/components/chat.py

```diff
@@ -56,52 +58,55 @@ def chat() -> rx.Component:
 
 def action_bar() -> rx.Component:
     """The action bar to send a new message."""
-    return rx.chakra.box(
-        rx.chakra.vstack(
+    return rx.center(
+        rx.vstack(
             rx.chakra.form(
                 rx.chakra.form_control(
-                    rx.chakra.hstack(
-                        rx.chakra.input(
-                            placeholder="Type something...",
-                            id="question",
-                            _placeholder={"color": "#fffa"},
-                            _hover={"border_color": styles.accent_color},
-                            style=styles.input_style,
+                    rx.hstack(
+                        rx.radix.text_field.root(
+                            rx.radix.text_field.input(
+                                placeholder="Type something...",
+                                id="question",
+                                width="30em",
+                            ),
+                            rx.radix.text_field.slot(
+                                rx.tooltip(
+                                    rx.icon("info", size=18),
+                                    content="Enter a question to get a response.",
+                                )
+                            ),
                         ),
-                        rx.chakra.button(
+                        rx.button(
                             rx.cond(
                                 State.processing,
                                 loading_icon(height="1em"),
                                 rx.chakra.text("Send"),
                             ),
                             type_="submit",
-                            _hover={"bg": styles.accent_color},
-                            style=styles.input_style,
                         ),
+                        align_items="center",
                     ),
                     is_disabled=State.processing,
                 ),
                 on_submit=State.process_question,
                 reset_on_submit=True,
-                width="100%",
             ),
             rx.chakra.text(
                 "ReflexGPT may return factually incorrect or misleading responses. Use discretion.",
-                font_size="xs",
-                color="#fff6",
                 text_align="center",
+                font_size=".75em",
```

Comment:
Should we switch to Radix text and use the `size` prop

Reply:
This needs to be smalled than the default 1 size, that why I had to resort to manually setting font size here